build.rs

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 fn main() {
-    if std::path::Path::new("/dev/sgx/enclave").exists() {
+    if std::path::Path::new("/dev/sgx_enclave").exists() {
```

Comment:
Should we maybe have `const SGX_ENCLAVE_DEVICE_PATH = ...;` with a docstring explaining why the path is what it is, how it gets populated, and how/whether it could be created dynamically? (This would be the place for the "hey your distro is broken, go fix your distro" comment")

Reply:
It looks like doing this will cause more pain than it solves because we have references to this path across compilation units with no shared code. Creating a shared crate for this problem is not worth the overhead.